src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/hmi/rc_on_remote_control_settings_notification.cc

```diff
@@ -99,6 +96,7 @@ void RCOnRemoteControlSettingsNotification::Run() {
     hmi_apis::Common_RCAccessMode::eType access_mode =
```

Comment:
`allowed` parameter is optional, the code above on line 84 to check the existence of `allowed` and return if not,  treats the parameter as mandatory. That's not correct.


```xml 
<function name="OnRemoteControlSettings" messagetype="notification">
  <description>Sender: vehicle -> RSDL. Notification about remote-control settings changed. Sent after User`s choice through HMI.</description>
  <param name="allowed" type="Boolean" mandatory="false" >
    <description>If "true" - RC is allowed; if "false" - RC is disallowed.</description>
  </param>
  <param name="accessMode" type="Common.RCAccessMode" mandatory="false" >
    <description>The remote control access mode specified by the driver via HMI.</description>
  </param>
</function>
```

Reply:
@yang1070 Fixed bf302e5d03f8e9eefa5af85bfbb8fd66fb3e6329